pynautobot/core/endpoint.py

```diff
@@ -232,13 +254,19 @@ def filter(self, *args, api_version=None, **kwargs):
             raise ValueError("A reserved {} kwarg was passed. Please remove it " "try again.".format(RESERVED_KWARGS))
 
         api_version = api_version or self.api.api_version
+
+        manual_pagination = self._check_manual_pagination(limit, offset)
+        url = "{}/".format(self.url)
+        if manual_pagination:
+            url += "?" + urllib.parse.urlencode({"limit": limit, "offset": offset})
```

Comment:
If this sticks around, let's look at passing it to the Requests library with `params`.